homeassistant/components/shelly/light.py

```diff
@@ -390,3 +395,39 @@ async def async_turn_on(self, **kwargs: Any) -> None:
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Turn off light."""
         await self.call_rpc("Switch.Set", {"id": self._id, "on": False})
+
+
+class RpcShellyLight(ShellyRpcEntity, LightEntity):
+    """Entity that controls a light on RPC based Shelly devices."""
+
+    _attr_color_mode = ColorMode.BRIGHTNESS
+    _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
+
+    def __init__(self, coordinator: ShellyRpcCoordinator, id_: int) -> None:
+        """Initialize light."""
+        super().__init__(coordinator, f"light:{id_}")
+        self._id = id_
+
+    @property
+    def is_on(self) -> bool:
+        """If light is on."""
+        return bool(self.coordinator.device.status[self.key]["output"])
```

Comment:
```
homeassistant/components/shelly/entity.py:        return cast(dict, self.coordinator.device.status[self.key])
homeassistant/components/shelly/entity.py:                self.coordinator.device.status[self.key].get(
homeassistant/components/shelly/entity.py:            self._last_value = self.coordinator.device.status[self.key][
homeassistant/components/shelly/entity.py:            self.coordinator.device.status[self.key][self.entity_description.sub_key]
homeassistant/components/shelly/light.py:        return bool(self.coordinator.device.status[self.key]["output"])
homeassistant/components/shelly/switch.py:        return bool(self.coordinator.device.status[self.key]["output"])
homeassistant/components/shelly/update.py:            self.coordinator.device.status[self.key][self.entity_description.sub_key],
```

It might make sense to make `self.coordinator.device.status[self.key]` a property so you don't have to write it out every time

Reply:
I already made this a property, no one knows why it is not used (first result in your search is the property) 😊